openedx/stanford/lms/djangoapps/instructor_analytics/basic.py

```diff
@@ -72,6 +72,10 @@ def student_responses(course):
                 elif 'student_answer' in state_dict:
                     raw_answer = state_dict['student_answer']
                     pretty_answers = u"{answer}".format(answer=raw_answer)
+                elif 'attempted' in state_dict and 'completed' in state_dict: # problem-builder
```

Comment:
I believe PEP8/pylint expect two spaces before the comment character `#`:

```python
foo = 'bar'  # problem-builder
```

Reply:
I wonder if we'll catch any other blocks w/ this...

Do you know if other components contain this values?

Reply:
That's what I've tried to avoid.